src/test/features/dashboard/routes/claimStatus/full-defence.ts

```diff
@@ -0,0 +1,271 @@
+    ]
+  },
+  {
+    status: 'Full defence - defendant dispute all of the claim and accepts mediation - defendant offers settlement to settle out of court - claimant rejected offer',
+    claim: fullDefenceClaim,
+    claimOverride: {
+      ...respondedAt,
+      response: {
+        ...baseResponseData,
+        ...baseDefenceData,
+        freeMediation: FreeMediationOption.YES
+      },
+      ...directionsQuestionnaireDeadline,
+      ...settlementOfferReject
+    },
+    claimantAssertions: ['000MC000', fullDefenceClaim.claim.defendants[0].name + ' has rejected the claim. They’ve suggested mediation to help resolve this dispute.','You’ve rejected the defendant’s offer to settle out of court. You won’t receive any more offers from the defendant.'],
+    defendantAssertions: ['000MC000', 'You have rejected the claim. You’ve suggested mediation.','The claimant has rejected your offer to settle the claim. Complete the directions questionnaire.']
+  },
+  {
+    status: 'Full defence - defendant dispute all of the claim and reject mediation - defendant offers settlement to settle out of court - claimant rejected offer',
+    claim: fullDefenceClaim,
+    claimOverride: {
+      ...respondedAt,
+      response: {
+        ...baseResponseData,
+        ...baseDefenceData
+      },
+      ...directionsQuestionnaireDeadline,
+      ...settlementOfferReject
+    },
+    claimantAssertions: ['000MC000', 'The defendant has rejected your claim','You’ve rejected the defendant’s offer to settle out of court. You won’t receive any more offers from the defendant.'],
+    defendantAssertions: ['000MC000', 'You’ve rejected the claim and said you don’t want to use mediation to solve it. You’ll have to go to a hearing.','The claimant has rejected your offer to settle the claim. Complete the directions questionnaire.']
+  },
+  {
+    status: 'Full defence - defendant dispute all of the claim - defendant offers settlement to settle out of court - claim settled with agreement',
+    claim: fullDefenceClaim,
+    claimOverride: {
+      ...respondedAt,
+      response: {
+        ...baseResponseData,
+        ...baseDefenceData
+      },
+      ...directionsQuestionnaireDeadline,
+      ...settledWithAgreement
+    },
+    claimantAssertions: ['000MC000', 'You’ve both signed a legal agreement. The claim is now settled.'],
+    defendantAssertions: ['000MC000', 'You’ve both signed a legal agreement. The claim is now settled.']
+  }
+]
+
+const claimPagePath = Paths.claimantPage.evaluateUri({ externalId: fullDefenceClaim.externalId })
+const defendantPagePath = Paths.defendantPage.evaluateUri({ externalId: fullDefenceClaim.externalId })
+
+describe('Dashboard page', () => {
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+    checkAuthorizationGuards(app, 'get', claimPagePath)
+    checkAuthorizationGuards(app, 'get', defendantPagePath)
```

Comment:
Aren't the default hooks and authorization guards already invoked for these pages elsewhere? This test file could be just for content.

Reply:
I don't think so, I can see all these tests are individual pages, it doesn't go with flow or user journeys